recipes-domd/domd-image-weston/files/meta-xt-prod-extra/recipes-extended/xen/xen_git.bbappend

```diff
@@ -25,7 +25,7 @@ SRC_URI_append_r8a7796 = " \
 # Generic
 ################################################################################
 
-FLASK_POLICY_FILE = "xenpolicy-${XEN_REL}.0"
+FLASK_POLICY_FILE = "xenpolicy-${XEN_REL}.0-rc"
```

Comment:
We currently have:
lrwxrwxrwx 2 xtfs xtfs        16 Jan 17 23:01 xenpolicy -> xenpolicy-4.10.0
so, just curios if this symlink be created properly

Reply:
Yes, the link is correct. On rc builds xenpolicy file has rc in its name.

Reply:
My point is that some of us might have their environment expecting the symlink without "rc" suffix...

Reply:
But it is the way XEN build system names xenpolicy file for rc builds.

Reply:
In xen/xsm/flask/Makefile you can find following:

POLICY_SRC := $(FLASK_BUILD_DIR)/xenpolicy-$(XEN_FULLVERSION)

Reply:
So it might have unstable/rc suffixes.